doc/user_guide/user_guide.md

```diff
@@ -288,11 +288,27 @@ final VirtualSchema virtualSchema=factory.createVirtualSchemaBuilder("THE_VIRTUA
         .dialectName("Exasol")
         .adapterScript(adapterScript)
         .connectionDefinition(connectionDefinition)
-        .properties(Map.of("IS_LOCAL","true"
-        "LOG_LEVEL","ALL"))
+        .properties(Map.of("IS_LOCAL", "true", "LOG_LEVEL", "ALL"))
         .build();
 ```
 
+#### Debug Output
+
+For virtual schemas Test DB Builder supports two special system properties to enable debug output to a remote machine:
+
+| System property | Sample value | Default value |
+|-----------------|--------------|---------------|
+| `com.exasol.virtualschema.debug.address` | `"192.168.1.2:3000"` | (none) |
+| `com.exasol.virtualschema.debug.level` | `"WARN"` | `"ALL"` |
+
+Sample usage
+```
+-Dcom.exasol.virtualschema.debug.address="1.2.3.4:3000"
+-Dcom.exasol.virtualschema.debug.level="INFO"
+```
+
+This enables to activate debug output in your local development environment without actually modifying any code and thereby preventing to commit debug settings by accident to your source code repository.
```

Comment:
Good idea.
We now probably have duplication in some of the VS, that used that trick already. Can you please do a code search?

Reply:
Done, tickets are listed in Team meeting minutes 2022-10-03